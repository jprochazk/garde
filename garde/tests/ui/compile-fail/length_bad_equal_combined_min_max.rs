#![allow(dead_code)]

#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(length(min = 1, max = 1, equal = 10))]
    min_max_equal: &'a str,
}

fn main() {}
