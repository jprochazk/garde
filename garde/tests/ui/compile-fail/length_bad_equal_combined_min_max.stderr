error: no `min` or `max` allowed if using `equal`
 --> tests/ui/compile-fail/length_bad_equal_combined_min_max.rs
  |
  |     #[garde(length(min = 1, max = 1, equal = 10))]
  |                    ^^^
